apps/spark/src/main/java/com/linkedin/openhouse/jobs/util/TableStatsCollectorUtil.java

```diff
@@ -79,11 +115,35 @@ public static IcebergTableStats populateStatsOfAllReferencedFiles(
   public static IcebergTableStats populateStatsForSnapshots(
       String fqtn, Table table, SparkSession spark, IcebergTableStats stats) {
 
-    Dataset<Row> dataFiles = getAllDataFilesCount(table, spark, MetadataTableType.FILES);
-    long countOfDataFiles = dataFiles.count();
-
-    // Calculate sum of file sizes in bytes in above allDataFiles
-    long sumOfFileSizeBytes = getSumOfFileSizeBytes(dataFiles);
+    Map<Integer, DataFilesSummary> currentSnapshotDataFilesSummary =
```

Comment:
same here

Reply:
thanks @maluchari 

I've pushed a second PR to address this: https://github.com/linkedin/openhouse/pull/270